integration_tests/e2e/activities/attendance/recordAttendance.cy.ts

```diff
@@ -45,23 +45,29 @@ context('Record attendance', () => {
 
   it('should click through record attendance journey', () => {
     const indexPage = Page.verifyOnPage(IndexPage)
-    indexPage.activitiesCard().should('contain.text', 'Activities, unlock and attendance')
     indexPage.activitiesCard().click()
 
     const activitiesIndexPage = Page.verifyOnPage(ActivitiesIndexPage)
-    activitiesIndexPage.recordAttendanceCard().should('contain.text', 'Record activity attendance')
     activitiesIndexPage.recordAttendanceCard().click()
 
     const recordAttendancePage = Page.verifyOnPage(AttendanceDashboardPage)
-    recordAttendancePage.recordAttendanceCard().should('contain.text', 'Record attendance and cancel activity sessions')
     recordAttendancePage.recordAttendanceCard().click()
 
     const selectPeriodPage = Page.verifyOnPage(SelectPeriodPage)
     selectPeriodPage.enterDate(new Date(today))
+    selectPeriodPage.selectAM()
     selectPeriodPage.continue()
 
     const activitiesPage = Page.verifyOnPage(ActivitiesPage)
-    activitiesPage.activityRows().should('have.length', 5)
+    activitiesPage.containsActivities('English level 1', 'English level 2', 'Football', 'Maths level 1')
+    activitiesPage.back()
+
+    Page.verifyOnPage(SelectPeriodPage)
+    selectPeriodPage.enterDate(new Date(today))
+    selectPeriodPage.selectPM()
```

Comment:
Select AM,  view activities,  click back link, select PM also and view activities